app/assets/stylesheets/styles.css

```diff
@@ -19,6 +19,7 @@
 @source '../../../app/decorators';
 @source '../../../app/javascript';
 @source '../../../app/views';
+@source not '../../../app/views/active_admin';
```

Comment:
This uses a Tailwind feature just added and not yet officially released ( https://github.com/tailwindlabs/tailwindcss/pull/ 17255 ), so we are moving to the latest Tailwind insiders release in this PR.